examples/python/arkitscenes/main.py

```diff
@@ -0,0 +1,180 @@
+    rotation = R.from_rotvec(np.asarray(angle_axis))
+
+    # Extract translation from the fifth to seventh tokens
+    translation = np.asarray([float(tokens[4]), float(tokens[5]), float(tokens[6])])
+
+    # Create tuple in format log_rigid3 expects
+    camera_from_world = (translation, rotation.as_quat())
+
+    return (ts, camera_from_world)
+
+
+def log_arkit(recording_path: Path) -> None:
+    """
+    Logs ARKit recording data using Rerun.
+
+    Args:
+        recording_path (Path): The path to the ARKit recording.
+
+    Returns
+    -------
+        None
+    """
+    video_id = recording_path.stem
+    image_dir = recording_path / "lowres_wide"
+    depth_dir = recording_path / "lowres_depth"
+    intrinsics_dir = recording_path / "lowres_wide_intrinsics"
+    traj_path = recording_path / "lowres_wide.traj"
+
+    depth_filenames = [x.name for x in sorted(depth_dir.iterdir())]
+    lowres_frame_ids = [x.split(".png")[0].split("_")[1] for x in depth_filenames]
+    lowres_frame_ids.sort()
+
+    # dict of timestamp to pose which is a tuple of translation and quaternion
+    poses_from_traj = {}
+    with open(traj_path, "r", encoding="utf-8") as f:
+        traj = f.readlines()
+
+    for line in traj:
+        ts, camera_from_world = traj_string_to_matrix(line)
+        # round timestamp to 3 decimal places
+        ts = f"{round(float(ts), 3):.3f}"
+        poses_from_traj[ts] = camera_from_world
+
+    rr.log_view_coordinates("world", up="+Z", right_handed=True, timeless=True)
+    ply_path = recording_path / f"{recording_path.stem}_3dod_mesh.ply"
+
+    # TODO(pablovela5620): for now just use the untextered/uncolored mesh until #1580 is resolved
+    mesh_ply = trimesh.load(str(ply_path))
+    mesh_gltf_bin = trimesh.exchange.gltf.export_glb(mesh_ply, include_normals=True)
+    rr.log_mesh_file("world/mesh", MeshFormat.GLB, mesh_gltf_bin, timeless=True)
+
+    bbox_annotations_path = recording_path / f"{recording_path.stem}_3dod_annotation.json"
+    annotation = load_json(bbox_annotations_path)
+    log_annotated_bboxes(annotation)
+
+    # To avoid logging image frames in the beginning that dont' have a trajectory
+    # This causes the camera to expand in the beginning otherwise
+    init_traj_found = False
+    for frame_id in tqdm(lowres_frame_ids):
+        rr.set_time_seconds("time", float(frame_id))
+        bgr = cv2.imread(f"{image_dir}/{video_id}_{frame_id}.png")
+        depth = cv2.imread(f"{depth_dir}/{video_id}_{frame_id}.png", cv2.IMREAD_ANYDEPTH)
+
+        # Log the camera transforms:
+        if frame_id in poses_from_traj:
+            if not init_traj_found:
+                init_traj_found = True
+            intrinsic_fn = intrinsics_dir / f"{video_id}_{frame_id}.pincam"
+            w, h, fx, fy, cx, cy = np.loadtxt(intrinsic_fn)
+            intrinsic = np.array([[fx, 0, cx], [0, fy, cy], [0, 0, 1]])
+            camera_from_world = poses_from_traj[str(frame_id)]
```

Comment:
```suggestion
            camera_from_world = poses_from_traj[frame_id]
```

Reply:
Also, `poses_from_traj` should probably be called something like `poses_from_timestamp` or something (depending on what you decide to call frame_id)